### ISSUE TITLE
openldap binaries fail due to missing .so file on nixos-unstable

### ISSUE THREAD

**Issue by womfoo** [NixOS/nixpkgs#15643]
### Steps to reproduce

```
$ ldapsearch -V
ldapsearch: error while loading shared libraries: libldap-2.4.so.2: cannot open shared object file: No such file or directory
```

## Technical details

* System: (NixOS: `nixos-version`, Ubuntu/Fedora: `lsb_release -a`, ...)
16.09.git.25fbac5 (Flounder)

* Nix version: (run `nix-env --version`)
nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion

* Nixpkgs version: (run `nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion`)
"16.09pre83955.cc41cb3"

**Comment by joachifm** [comment]
@womfoo I think this has been resolved by 92ce529880c5d7622f6469ac1395f120fcb749d4; can you confirm?

**Comment by womfoo** [comment]
That commit indeed fixes the issue. Thanks @joachifm
